js/spotify.js

```diff
@@ -1,12 +1,48 @@
-// API Docs at: 
+// API Docs at:
 // https://developer.spotify.com/web-api/search-item/
+$(document).ready(function () {
 
+  $('#search-type artist').on('click', function(e) {
+    var artists = $(this).text();
 
-function searchByArtist(keyword) {
-  var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
-}
+    searchByArtist(artists);
+    e.preventDefault();
+  })
+  searchByArtist('#results');
 
+  function searchByArtist(keyword) {
+    var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
+    //console.log();
+  }
 
-function searchByTrack(keyword) {
-  var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=track';
-}
\ No newline at end of file
+  $('#search-type track').on('click', function(e) {
+    var tracks = $(this).text();
+
+    searchByTrack(tracks);
+    e.preventDefault();
+  })
+  searchByTrack('#results')
+
+
+  function searchByTrack(keyword) {
+    var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=track';
+    // console.log();
+  }
+
+  var request = $.ajax(options);
+
+  request.done(function(output) {
+  })
+
+  equest.fail(function(jqXHR, status, errorThrown) {
+    console.log('Error', errorThrown);
+  });
+
+  request.always(function() {
+    console.log('Always');
+  });
+
+
+searchByArtist();
```

Comment:
Remember these two functions that you created both take arguments to work so you have to pass it something for it not to return undefined.